src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -18,18 +18,18 @@ A new web platform API called [Portals](https://github.com/WICG/portals) can hel
 <img class="w-screenshot" src="hero.gif" alt="Showing a demo of seamless navigations between two sites using Portals">
 
 ## What Portals enables
-**Portals** is a new HTML element that can help you keep your front-end simple while allowing seamless navigations with custom transitions. Like dynamic navigations in Single Page Applications with an [App Shell model](https://developers.google.com/web/fundamentals/architecture/app-shell),
+Portals are new HTML elements that can help you keep your front-end simple while allowing seamless navigations with custom transitions. Like dynamic navigations in Single Page Applications with an [App Shell model](https://developers.google.com/web/fundamentals/architecture/app-shell),
```

Comment:
"Like dynamic navigations in Single Page Applications with an App Shell model, Portals let you dynamically change content with transitions while keeping your Multiple Page Application architecture."

I don't know if this consolidation quite makes sense. The original text did a bit more to explain the pros/cons of single and multi page experiences and how portals bridged them.

Reply:
I had a totally different reaction to the original text, in that I felt like it wasn't making the connection between SPA and MPA clear enough, more just explaining what these were. But that doesn't mean we aren't quite there yet. I can have another go to try and find the happy middle.

Reply:
yeah I think maybe the gist is something like... SPAs offer nice transitions but come at the cost of higher complexity to build. MPAs are much easier to build, but you end up with blank screens between pages. Portals offer the best of both worlds—the low complexity of an MPA with the seamless transitions of a SPA.

Reply:
Done. Much better.